### ISSUE TITLE
npm does not install latest release

### ISSUE THREAD

**Issue by ErikAGriffin** [zohararad/audio5js#78]
Hi, I'm not quite sure how our versioning works or how this repository relates to the npm package, but the npm audio5 package is out of date.  If you instruct me how to do so I can update it myself with the latest src

**Comment by zohararad** [comment]
Hey @ErikAGriffin . Latest version - v0.1.10 is also the latest version on npmjs and is in sync with master. Where are you finding incompatibilities?

**Comment by ErikAGriffin** [comment]
I did a diff of the non-minified source from the github repo versus the non-minified source downloaded via npm and there were a lot of differences.
I spotted my own one line contribution in the diff so I concluded the npm package must not be updated.

On tor., apr. 7, 2016 at 8:58 p.m., zohararad <notifications@github.com> wrote:
Hey @ErikAGriffin [https://github.com/ErikAGriffin] . Latest version - v0.1.10 is also the latest version on npmjs and is in sync with master. Where are you finding incompatibilities?

**Comment by zohararad** [comment]
ok, this is a little strange. I can push master as v0.1.11 - will that help?

**Comment by ErikAGriffin** [comment]
I'm not sure, but if I suppose it won't hurt. As I said I am unfamiliar with npm packaging so I'll do some research. Where is the versioning managed in the audio5 repo?

On tor., apr. 7, 2016 at 9:11 p.m., zohararad <notifications@github.com> wrote:
ok, this is a little strange. I can push master as v0.1.11 - will that help?

**Comment by zohararad** [comment]
Nope.. just me. I suppose you could check out master and v0.1.10 and diff them - I did that and found no diffs. Not sure how NPM packages stuff.

Also, you can always tell NPM to install from Github as far as I know.

**Comment by ErikAGriffin** [comment]
True, but for users who just want to dive in I think it's important they can simply use npm to grab the latest version of audio5.
I'll dig into node and see what I can find out

On tor., apr. 7, 2016 at 9:17 p.m., zohararad <notifications@github.com> wrote:
Nope.. just me. I suppose you could check out master and v0.1.10 and diff them - I did that and found no diffs. Not sure how NPM packages stuff.

Also, you can always tell NPM to install from Github as far as I know.
